js/data.js

```diff
@@ -6,3 +6,12 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
+window.addEventListener('beforeunload', function (event) {
+  var jsonData = JSON.stringify(data);
+  this.localStorage.setItem('jsonData', jsonData);
+});
+
+if (localStorage.getItem('jsonData') === JSON.stringify(data)) {
+  data = JSON.parse(JSON.stringify(data));
```

Comment:
We want data to be converted back to an object. It has already been stringified on line 11. We now want to parse the string we are getting from local storage.
`data = JSON.parse(localStorage.getItem('jsonData'));`

Reply:
thank you! was trudging my way through these last couples lines of code